### DISCUSSION TITLE
Practiva Terminada de PAsteleria

### DISCUSSION THREAD

**Discussion by DaniielGarrido** [Launch-X-Latam/MisionFrontEnd#333] [General]
Espero qeue asi sea la practica porqe sino lo es ya no tengo tiempo de componer nada 

link de pag.
https://daniielgarrido.github.io/Pasteleria.github.io/index.html

link de repositorio 
https://github.com/DaniielGarrido/Pasteleria.github.io

Exito a tosdos

**Comment by DaveOval** [comment]
me gusta mucho, super completa y limpia
